lib/ContextualPopup/ContextualPopup.js

```diff
@@ -127,6 +127,12 @@ module.exports = kind(
 		*/
 		spotlightModal: false,
 
+		/* @type {Boolean}
+		* @default false
+		* @public
+		*/
+		modal: false,
```

Comment:
Is this needed, if the default value is also `false` in `enyo/Popup`? Also, we should consider cleaning up the documentation here if we choose to keep it (adding a description, and not placing any pragmas on the opening line of the comment block, which should begin with /** instead of /*).